R/tables_spark.R

```diff
@@ -0,0 +1,73 @@
+tbl_quote_name <- function(name) {
+  x <- gsub("`", "``", name, fixed = TRUE)
```

Comment:
Just double checking -- the intention here is to quote a table name e.g.

    foo`bar

as

    `foo``bar`

?